model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -66,7 +81,8 @@ def set_last_results(self, measurement):
         ----------
         measurement: Measurement from the last run
         """
-        pass
+        self._last_results = measurement
+        self._all_results.extend(measurement)
```

Comment:
Why not use `.append`? Isn't measurement just a single value?

Reply:
as we discussed, it is a list. I have updated the docstrings and variable name